src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -2,15 +2,15 @@
 title: "Is your app installed? getInstalledRelatedApps() will tell you!"
 subhead: |
   "The `getInstalledRelatedApps()` method allows your web app to check whether
-  your native app or PWA is installed on a user's device."
+  your app or PWA is installed on a user's device."
```

Comment:
```suggestion
  your Android app or PWA is installed on a user's device."
```

Reply:
Please make this change throughout.

Reply:
Currently it's true, but it's possible Microsoft will allow sites to check if a Windows app is installed, etc. So I left it generic, except for the pieces that are specific to Android apps.